Sources/TensorFlow/Operators/NN.swift

```diff
@@ -384,50 +384,48 @@ func _vjpAvgPool3D<Scalar: TensorFlowFloatingPoint>(
     })
 }
 
-/// Computes a 2-D convolution using `self` as input, with the specified
-/// filter, strides, and padding.
+/// Computes a 2-D convolution with the specified input, filter, strides, and padding.
 ///
 /// - Parameters:
-///   - x: The input.
+///   - input.
```

Comment:
This is not the correct documentation comment syntax. [Swift API Design Guidelines - Fundamentals](https://swift.org/documentation/api-design-guidelines/#fundamentals) which describes the documentation comment syntax.

You should add a colon after the _parameter name_, followed by a _noun phrase_ that describes the parameter, followed by a period.
```suggestion
///   - input: The input.
```

Reply:
@rxwei alright, i just find this very redundant is all. I'll update it accordingly.